bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/i18n/tapocontrol.properties

```diff
@@ -163,3 +163,64 @@ channel-type.tapocontrol.effectName.label = Effect Name
 channel-type.tapocontrol.effectName.description = Name of LightningEffect
 channel-type.tapocontrol.effectOn.label = Lightning Effect Enable
 channel-type.tapocontrol.effectOn.description = Switches the lightning effect on/off
+
+# error messages
+
+error-api-account = received error code (-2101);
+error-api-aes-decode-fail = aes decode failed (-1005)
+error-api-antitheft-conflict = received error code (-2002);
```

Comment:
Why defining error messages if finally the message is always the same with only difference a number ?
In that case, you just need one entry with a parameter.